### PR TITLE
11.0 mig report mrp bom matrix

### PR DESCRIPTION
**Pull request by mpanarin**
Migrating module to version 11

### CODE REVIEW

**Review by max3903 — APPROVED** [review]

File: report_mrp_bom_matrix/views/report_mrp_bom_matrix_view.xml [hunk 1]
```diff
@@ -32,7 +32,7 @@
         <field name="name">report.mrp.bom.matrix.pivot</field>
         <field name="model">report.mrp.bom.matrix</field>
         <field name="arch" type="xml">
-            <pivot string="Report MRP BOM Matrix" create="0" delete="0">
```
Comment (JordiBForgeFlow) [inline report_mrp_bom_matrix/views/report_mrp_bom_matrix_view.xml]: Why do you replace the create and delete options? The bom matrix is intended to be a report and we do not expect anyone to be able to create records.


image

File: report_mrp_bom_matrix/README.rst [hunk 2]
```diff
@@ -19,7 +19,7 @@ out of it.
 Usage
 =====
 
-* Go to 'Manufacturing / Products / BOM Matrix'.
+* Go to 'Manufacturing / Master Data / BOM Matrix'.
```
Comment (JordiBForgeFlow) [inline report_mrp_bom_matrix/README.rst]: You should better move the menu to 'Manufacturing / Reporting / BOM Matrix', as it is really a report.

**Review by JordiBForgeFlow — CHANGES_REQUESTED** [review]

**Comment by JordiBForgeFlow** [comment]
@mpanarin I have added to comments to your PR.

**Comment by pedrobaeza** [comment]
Why ` report_mrp_bom_matrix`? It should be according naming conventions `mrp_bom_matrix_report`.

**Comment by JordiBForgeFlow** [comment]
@pedrobaeza Agreed. @mpanarin Will you change that too everywhere? From module name to the model name.

**Comment by mpanarin** [comment]
@jbeficent sure

File: report_mrp_bom_matrix/views/report_mrp_bom_matrix_view.xml (hunk 1 above)
Comment (mpanarin) [reply report_mrp_bom_matrix/views/report_mrp_bom_matrix_view.xml]: pivots don't support `create` and `delete`. Moved them to form view.

File: report_mrp_bom_matrix/README.rst (hunk 2 above)
Comment (mpanarin) [reply report_mrp_bom_matrix/README.rst]: done

**Review by JordiBForgeFlow — APPROVED** [review]

**Pull request merged**